ti/savings/savings.go

```diff
@@ -43,4 +46,22 @@ func ParseAndUploadSavings(ctx context.Context, workspace string, log *logrus.Lo
 
 	// DLC Savings (Placeholder)
 	// Cache Intel savings (Placeholder)
+	return getStepState(states)
```

Comment:
Don't understand it, Why is states an array/list?

Reply:
There are 4 features: Build Cache, TI, DLC and Cache Intel. Each feature will give us a state and `getStepState` helps us get the final state of the step from all the feature states.